core-bundle/tests/Cache/ContaoCacheWarmerTest.php

```diff
@@ -73,6 +73,7 @@ public function testCreatesTheCacheFolder(): void
         $this->assertFileExists(Path::join($this->getTempDir(), 'var/cache/contao/config/autoload.php'));
         $this->assertFileExists(Path::join($this->getTempDir(), 'var/cache/contao/config/config.php'));
         $this->assertFileExists(Path::join($this->getTempDir(), 'var/cache/contao/config/templates.php'));
+        $this->assertFileExists(Path::join($this->getTempDir(), 'var/cache/contao/config/available-language-files.php'));
```

Comment:
Should we also add a test that verifies the contents of the *available-language-files.php* file?